### PR TITLE
12 feature double scroll left carousel and spiral

### PR DESCRIPTION
**Pull request by ghost**
[ISSUE](https://github.com/)

## Context
added scrollProvider and left-bar styled.

## Pull request checklist

- [x] Commit message is descriptive
- [x] Make a single commit
- [ ] Folder structure & File names meets naming conventions
- [ ] Responsive (Mobile)
- [ ] According to design

### CODE REVIEW

File: providers/ScrollProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+interface ScrollProviderProps {
+  children: ReactNode;
+}
+
+export const ScrollProvider: FC<ScrollProviderProps> = ({ children }) => {
+  const [scroll, setScroll] = useState<number>(0);
+
+  useEffect(() => {
+    window.addEventListener("mousewheel", (e: Event) => {
```
Comment (Enhmunh-E) [inline providers/ScrollProvider.tsx:25]: onscroll bishu 0-0 touchpadtai ajilhu

**Review by Enhmunh-E — CHANGES_REQUESTED** [review]

**Comment by jangar715** [comment]
ajlaadeleshd

**Comment by Enhmunh-E** [comment]
conflict bro

File: providers/ScrollProvider.tsx (hunk 1 above)
Comment (Akiko643) [reply providers/ScrollProvider.tsx]: ajiljiisan2

**Review by Enhmunh-E — DISMISSED** [review]

**Review by Enhmunh-E — APPROVED** [review]

**Pull request merged**